src/options/options_template.cpp

```diff
@@ -435,7 +435,7 @@ void Options::parseOptionsRecursive(Options* options,
   }
 
   // Having this synonym simplifies the generation code in mkoptions.
-  options::OptionsHandler* handler = options->d_handler;
+  options::OptionsHandler* d_handler = options->d_handler;
```

Comment:
Why this name change? `d_handler` is not a data member of Options is it?

Reply:
`mkoptions.py` uses the same template in various places. In one place, the handler is called `d_handler`...
I can check whether I can improve on this...

Reply:
This weirdness was required solely because `parseOptionsRecursive` was a static method for ... no particular reason at all. :-)